grails-app/views/stockMovement/show.gsp

```diff
@@ -206,6 +206,14 @@
                                 ${stockMovement?.stocklist?.name?:warehouse.message(code:"default.none.label")}
                             </td>
                         </tr>
+                        <tr class="prop">
+                            <td class="name">
+                                <warehouse:message code="stockMovement.approvers.label"/>
+                            </td>
+                            <td class="value">
+                                ${stockMovement?.requisition?.approvers?.join(", ") ?: g.message(code:"default.none.label")}
```

Comment:
Do you want to display all approvers? Perhaps it would be better to use "ellipsis" and display all only on hover, or `+x` for those that don't fit in space